assets/js/components/notifications/ThankWithGoogleSupporterWallNotification.js

```diff
@@ -47,24 +47,25 @@ export default function ThankWithGoogleSupporterWallNotification() {
 	return (
 		<BannerNotification
 			id="twg-supporter-wall-prompt"
+			className="googlesitekit-twg-supporter-wall-banner"
 			title={ __(
 				'Add a Thank with Google supporter wall',
 				'google-site-kit'
 			) }
 			description={
 				<Fragment>
-					<p>
```

Comment:
Why was this changed to use a `span` in place of a `p` tag? Paragraph would be more appropriate for this case.

Reply:
I have changed it due to the React warning that I mentioned in the PR description.

![Screenshot 2022-10-04 at 9 49 13 PM](https://user-images.githubusercontent.com/24408261/194105890-739bbe76-b130-48f3-bfb5-d50be71eabb3.png)

The error is caused since the `description` is rendered within the `<p>` tag in the `BannerNotification`. However, the warning appears when we try to nest the strings with the `<p>` tag. Hope that makes sense.

Reply:
Ah I see. I don't think we should be rendering the `description` in a wrapping `p` tag when we pass an element but that's out of scope here. Thanks for clarifying.

Reply:
I initially used `<p>` tags since, as per the Figma design, we have to render the message as two different paragraphs, not one.

Reply:
I opened #5972 to address this.